guides/common/modules/ref_dhcp-options-for-network-configuration.adoc

```diff
@@ -27,4 +27,8 @@ Set this to the address for {ProjectServer} on `eth1`.
 --foreman-proxy-dhcp-server::
   Sets the address of the DHCP server to manage.
 
+--foreman-proxy-dhcp-subnets::
+  Sets the subnets of the DHCP server to manage.
```

Comment:
Should this mention that by default all subnets are managed? This helps to speed up things on large DHCP servers with many subnets, but the installer by default only supports a single subnet so I'm not sure this adds anything for most users. It'll only confuse them.

Reply:
I don't really know. Maybe someone else can weigh in?

Reply:
@ekohl What is the purpose of this argument in the first place? What is the scenario in which the user would prefer to use it?

Reply:
Here is the example:
https://github.com/theforeman/smart-proxy/blob/9268440e7ea33d3fd0cae22896c403338e049cb9/config/settings.d/dhcp.yml.example#L11-L12
It mostly benefits the `dhcp_native_ms` provider (which the installer doesn't manage). I'm looking through the code, but can't find the code that actually ensures the whole setting is used.

I'd say that for most, if not all, users of the installer this parameter is not needed. Documenting it here will confuse users and make them do additional work. My suggestion is to leave it undocumented.

Reply:
Maybe I can just put it behind `ifdef::orcharhino[]`?

Reply:
What is the benefit of including it in the Orcharhino manual? I'm quite sure it only affects `dhcp_native_ms` and that requires running on Windows. Has Orcharhino ported the installer to Windows?

I never looked at this before, but now as an installer maintainer I'm tempted to remove the option.

Reply:
Unless any product handles Windows, I'd go with undocumented.

Reply:
I have already removed anything regarding Windows for now. I'd be happy to push this upstream once a working installer exists.

Reply:
Then please remove this part. I strongly insist on it.

Reply:
OK, changed from if not satellite to if orcharhino.